obsRecorder.js

```diff
@@ -18,6 +19,23 @@ function initialize(webContents) {
   setupSources();
   obsInitialized = true;
 
+  const displayId = 'display1';
+  const displayWidth = 960;
+  const displayHeight = 540;
+
+  var cameraWindow = new BrowserWindow({
+    width: displayWidth,
+    height: displayHeight,
+  });
+
+  osn.NodeObs.OBS_content_createSourcePreviewDisplay(
+    cameraWindow.getNativeWindowHandle(),
+    "", // or use camera source Id here
+    displayId,
+  );
+  osn.NodeObs.OBS_content_setShouldDrawUI(displayId, false);
+  osn.NodeObs.OBS_content_resizeDisplay(displayId, displayWidth, displayHeight);
```

Comment:
In some cases (HiDPI displays and scaled UI) video width and height here will produce video that doesn't fill window completely (or overflows it). Because BrowserWindow is being created with virtual scaled pixels but here we specify hardware pixels.

So scaling logic from `setupSources` should also be applied here.